constructor/nsis/main.nsi.tmpl

```diff
@@ -1776,11 +1895,11 @@ Section "Uninstall"
     ${Else}
         StrCpy $R0 "system"
     ${EndIf}
-    # When running conda.bat directly, there is a non-fatal error
-    # that DOSKEY (called by conda_hook.bat) is not a valid command.
-    # While the operation still succeeds, this error is confusing.
-    # Calling via cmd.exe fixes that.
-    push '"$CMD_EXE" /D /C "$INSTDIR\condabin\conda.bat" init cmd.exe --reverse --$R0'
+    ${If} ${Silent}
+        push '"$INSTDIR\condabin\conda.bat" init cmd.exe --reverse --$R0 > "${STEP_LOG}" 2>&1'
+    ${Else}
+        push '"$INSTDIR\condabin\conda.bat" init cmd.exe --reverse --$R0'
+    ${EndIf}
```

Comment:
We don't need the CMD workaround anymore here because all commands are run via CMD anyway, right?